### PR TITLE
Furka - Pablo UI - Remove Constant Product Pools and Liquidity Bootstrapping Pools

### PR DESCRIPTION
**Pull request by saadjhk**
## Issue
Pablo was integrated with old DEX pools.

## Checklist

- [X] Remove dependency on `ConstantProductPool` and `LiquidityBootstrappingPool`.
- [X] Remove all previous integrations for `LiquidityBootstrappingPool`.
- [X] Build and CI checks
- [X] Comments to highlight where future integrations will be needed.

### CODE REVIEW

File: frontend/apps/pablo/defi/hooks/swaps/useSwaps.ts [hunk 1]
```diff
@@ -129,13 +129,13 @@ export function useSwaps({ selectedAccount }: { selectedAccount?: InjectedAccoun
...
   useEffect(() => {
-    if (constantProductPools.length > 0) {
-      const pool = constantProductPools.find((_constantPool) => {
-        const pair = _constantPool.getPair();
-        const pairBase = pair.getBaseAsset().toString();
-        const pairQuote = pair.getQuoteAsset().toString();
+    if (liquidityPools.length > 0) {
+      const pool = liquidityPools.find((_constantPool) => {
+        const pair = Object.values(_constantPool.getAssets().assets);
+        const pairBase = pair[0].toString();
```
Comment (andresmechali) [inline frontend/apps/pablo/defi/hooks/swaps/useSwaps.ts:137]: Will pair always be defined?

File: frontend/apps/pablo/defi/hooks/swaps/useSwaps.ts [hunk 2]
```diff
@@ -183,13 +183,14 @@ export function useSwaps({ selectedAccount }: { selectedAccount?: InjectedAccoun
...
+  const poolAssets = selectedPool ? Object.keys(selectedPool.getAssets().assets) : null;
+  let poolQuoteBalance = poolAssets ? 
+    poolAssets[0] === selectedAssetOneId
       ? quoteAmount
       : baseAmount
     : new BigNumber(0);
-  let poolBaseBalance = selectedPool
-    ? selectedPool.getPair().getQuoteAsset().toString() === selectedAssetOneId
+  let poolBaseBalance = poolAssets
+    ? poolAssets[1] === selectedAssetOneId
```
Comment (andresmechali) [inline frontend/apps/pablo/defi/hooks/swaps/useSwaps.ts:193]:
```suggestion
    ? poolAssets?.[1] === selectedAssetOneId
```

File: frontend/packages/shared/defi/lib/utils.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+export type Period = {
+    days?: number;
+    years?: number;
+    months?: number;
+    weeks?: number;
+};
+
+export function createDurationPresetLabel(period: Period): string {
+    let suf = period.days
```
Comment (andresmechali) [inline frontend/packages/shared/defi/lib/utils.ts:18]: Maybe a switch / case or some ifs here would be more readable ?

File: frontend/packages/shared/defi/lib/utils.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            : period.months
+                ? period.months.toFixed(2)
+                : period.years
+                    ? period.years.toFixed(2)
+                    : "-"
+        } ${suf}`;
+}
+
+export function calculatePeriod(durationInSeconds: string | number): Period {
+    return durationInSeconds < 7 * (DAYS / 1000) ?
```
Comment (andresmechali) [inline frontend/packages/shared/defi/lib/utils.ts:49]: Also here

**Review by andresmechali — APPROVED** [review]

File: frontend/apps/pablo/defi/hooks/swaps/useSwaps.ts (hunk 1 above)
Comment (saadjhk) [reply frontend/apps/pablo/defi/hooks/swaps/useSwaps.ts]: yup @andresmechali as long as pool data is correct from chain

File: frontend/packages/shared/defi/lib/utils.ts (hunk 3 above)
Comment (saadjhk) [reply frontend/packages/shared/defi/lib/utils.ts]: @andresmechali  cleaned

**Review by easteregg — APPROVED** [review]

**Pull request merged**